build.gradle

```diff
@@ -95,6 +99,8 @@ subprojects {
 
         verify_saml_test (
                 "uk.gov.ida:common-test-utils:${utils_version}",
+                "net.sourceforge.htmlunit:htmlunit:2.28",
```

Comment:
This doesn't look right. Why do we need to pull in `htmlunit`? Also why is `common-test-utils` being pulled in with two version numbers?

Reply:
Yep, that isn't right.

Reply:
This happened when I rebased